lighthouse-service/go.mod

```diff
@@ -10,9 +10,9 @@ require (
 	github.com/google/uuid v1.1.1
 	github.com/json-iterator/go v1.1.9 // indirect
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/keptn/go-utils v0.7.1
+	github.com/keptn/go-utils v0.6.3-0.20200923082506-f715579724ec
```

Comment:
What's going on here, why are we adding go-utils in version 0.6.3?

Reply:
this is the latest commit ID. Not sure why it starts with 0.6.3, but this is generated by go modules when targeting a specific commit